conf/ConfigOptions.asciidoc

```diff
@@ -1945,6 +1945,16 @@ sequencing of element ID's.
 List of element attributes to remove with the RemoveAttributeVisitor.  See ElementAttributeType for
 valid values.
 
+=== script.test.max.exec.time
+
+* Data Type: int
+* Default Value: `-1`
+
+For script test debugging only.  Sets a maximum allowed time, in seconds, for a script test to run.
+If the script runs longer than the specified time, then it is forcefully stopped by the system.  If
+the value is set to -1, then there is no time limit for script tests.  This is useful when
+debugging tests which may hang on a remote build server.
+
```

Comment:
I'm a big fan of this idea so will @sputnam. Hanging Jenkins jobs es no bueno.  I approve this twice.